zerver/tests/test_auth_backends.py

```diff
@@ -2601,10 +2601,6 @@ def test_social_auth_saml_login_bad_idp_arg(self) -> None:
             )
             self.assert_in_success_response(["Configuration error", "SAML authentication"], result)
 
-            result = self.client_get(f"/accounts/{action}/social/saml/")
-            # No matching URL pattern.
-            self.assertEqual(result.status_code, 404)
-
```

Comment:
Hmm, why would we want to delete this?

Reply:
Since `/accounts/{action}/social/saml/` and `/accounts/{action}/social/saml` both are acceptable URLs now.

Reply:
@amanagr Ah, although ` /accounts/{action}/social/saml/` (or without the slash) is never a valid url - the correct scheme is ` /accounts/{action}/social/saml/{idp_name}` (the IdP has to be specified - because we can have many SAML IdPs configured on the same deployment). So this should still be returning `404` I think.